packages/marko-web/src/express/index.js

```diff
@@ -14,6 +14,7 @@ module.exports = (config = {}) => {
   const distDir = path.resolve(rootDir, 'dist');
   const app = express();
   const serverDir = path.resolve(rootDir, 'server');
+  const { siteName: name } = config.coreConfig;
```

Comment:
Do be a bit more clear, don't rename `siteName` to name, and instead just send `{ name: siteName }` to apollo